react/ProductSummaryContext.tsx

```diff
@@ -115,11 +115,11 @@ const buildProductQuery = ((product: Product) => {
   return querystring.stringify(query)
 })
 
-function ProductSummaryProvider({ product, selectedItem, children }) {
+function ProductSummaryProvider({ product, selectedItem, isLoading, children }) {
   const initialState = {
     product,
     isHovering: false,
-    isLoading: false,
+    isLoading: Boolean(isLoading),
```

Comment:
Why is this necessary? Can't you just pass the `isLoading` value?

Reply:
It is not necessary 🤦 
Removed!